readme-vars.yml

```diff
@@ -25,11 +40,16 @@ param_env_vars:
 param_usage_include_ports: false
 param_usage_include_net: true
 param_net: "host"
-param_net_desc: "Shares host networking with container. Required for some devices to be discovered by Home Assistant."
+param_net_desc: "Shares host networking with container."
```

Comment:
I see no need for this change, the original text is accurate.

Reply:
You requested that this info be in a breakout section https://github.com/linuxserver/docker-homeassistant/pull/12#issuecomment-868858261

Reply:
I prefer to keep both for the lazy users that won't scroll down. Letting them know up front that not using host is a breaking change is pretty important for our support sanity.

Reply:
added it back